### DISCUSSION TITLE
VS code configuration

### DISCUSSION THREAD

**Discussion by diogo-garvis** [pycaret/pycaret#3425] [Q&A]
Guys, hi.
I have a small issue but I still didn't figure out how to solve it. I am running Pycaret in VSCODE (I have all Jupyter extensions installed). I am running a jupyter kernel but I cannot properly configure to show the markdown table (it only shows object name (pic attached). Has anyone had this issue? Thanks in advance.

<img width="614" alt="image" src="https://user-images.githubusercontent.com/108800127/227048978-7b0fde28-0e52-43b9-9ff0-c2e67f9ea30f.png">

**Comment by ngupta23** [comment]
Can you try 2 things?

(1) The same code in Jupyter Notebook or Jupyter Lab? If it works, it may be a VSCode-specific bug that you may have to file in their repo.
(2) Try in VSCode for the regression or classification module in pycaret. If it works, then it may be a TimeSeries-related bug in Pycaret.

Please report your finding back.

Thanks,
Nikhil

**Reply by turkalpmd** [reply]
Jupyter rendered result but VSCode cant.
